File: JS/MakeTheStringGreat.test.js
/**
 * @version JAVASCRIPT ECMAScript 6 
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
import TEST from 'tape'
import { makeGood } from './MakeTheStringGreat.js'

TEST('Starting MakeTheStringGreat test...', (t) => {
    //Test 1
    let want = "leetcode"
    let got = makeGood( "leEeetcode" )
    t.assert( want === got, "Expect: "+want)
    //Test 2
    want = ""
    got = makeGood( "abBAcC" )
    t.assert( want === got, "Expect: "+want)
    //Test 3
    want = "s"
    got = makeGood( "s" )
    t.assert( want === got, "Expect: "+want)

    t.end()
})

File: JAVA/MakeTheStringGreatTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class MakeTheStringGreatTest {

    @Test
	public void testMakeTheStringGreat() throws Exception {
        //Test 1
        String want = "leetcode";
        String got = MakeTheStringGreat.makeGood( "leEeetcode" );
        assertEquals(want, got);
        //Test 2
        want = "";
        got = MakeTheStringGreat.makeGood( "abBAcC" );
        assertEquals(want, got);
        //Test 3
        want = "s";
        got = MakeTheStringGreat.makeGood( "s" );
        assertEquals(want, got);
	}

}
